slither/detectors/operations/unused_return_values.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module detecting unused return values from external calls
+"""
+
+from collections import defaultdict
+from slither.detectors.abstract_detector import AbstractDetector, DetectorClassification
+from slither.slithir.operations.high_level_call import HighLevelCall
+
+class UnusedReturnValues(AbstractDetector):
+    """
+    If the return value of a function is never used, it's likely to be bug
+    """
+
+    ARGUMENT = 'unused-return'
+    HELP = 'Unused return values'
+    IMPACT = DetectorClassification.LOW
```

Comment:
Open to any feedback here on the impact/confidence. Went with a low/medium but was on the fence.